src/MuTalk-Model/MTAnalysis.class.st

```diff
@@ -265,6 +280,13 @@ MTAnalysis >> run [
 			  false ]
 ]
 
+{ #category : 'starting' }
+MTAnalysis >> startBudget [
+
+	budget ifNil: [ budget := self defaultBudget ].
+	budget start
```

Comment:
Can we "restart" a budget?

Reply:
First, the only budget that really needs to be started is the time budget. I could add a comment to explain it here.
But yes a budget can be restarted, it will just overwrite its initial time but I don't see this as a problem (I never tested it though, maybe it's worth writing a test for this). It's just that if it's restarted mid-analysis, overall the analysis will run longer.